PYTHON/DataTypes/use_userdatafunctioncontext.py

```diff
@@ -0,0 +1,15 @@
+# This sample uses a parameter with data type UserDataFunctionContext,
+# which is a parameter that contains certain metadata about the function
+# invocation, such as the invocation id and some properties of the token
+# used to invoke the function.
+
+import datetime
+
+@udf.context(argName="udfContext")
+@udf.function()
+def hello_fabric_with_userdatafunctioncontext(udfContext: fn.UserDataFunctionContext)-> str:
```

Comment:
@marshall-lerner  can we change hello_fabric_with_userdatafunctioncontext to get_function_invocation_details since its related to function execution